vector/src/main/java/im/vector/riotx/features/home/room/filtered/FilteredRoomsActivity.kt

```diff
@@ -58,12 +56,10 @@ class FilteredRoomsActivity : VectorBaseActivity() {
             }
 
             override fun onQueryTextChange(newText: String): Boolean {
-                // TODO Create a viewModel and remove this public fun
```

Comment:
This TODO was not a good idea?

Reply:
I think it's ok to handle it with a public fun, but I can put it back if you want

Reply:
ok to remove the TODO